documentation-site/examples/accordion/custom.js

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
linting fails on this file due to the error below:
```
throw new Error(`File ${file} is missing '// @flow' header comment.`);
```

```suggestion
// @flow
import * as React from 'react';
```